extensions/sql-database-projects/src/controllers/projectController.ts

```diff
@@ -425,6 +425,34 @@ export class ProjectsController {
 		await vscode.commands.executeCommand(constants.revealFileInOsCommand, vscode.Uri.file(project.projectFilePath));
 	}
 
+	/**
+	 * Opens the .sqlproj file for the given project. Upon update of file, prompts user to
+	 * reload their project.
+	 * @param context a treeItem in a project's hierarchy, to be used to obtain a Project
+	 */
+	public async editProjectFile(context: BaseProjectTreeItem): Promise<void> {
```

Comment:
can there be a test to verify that the reloaded project is updated as expected?

Reply:
may be add an dummy sql file and see its loaded in tree